packages/project-access/src/types/info/index.ts

```diff
@@ -1 +1,12 @@
+import type { CapProjectType } from '../cap';
+
 export type AppProgrammingLanguage = 'JavaScript' | 'TypeScript' | '';
+
+export type ProjectType = 'EDMXBackend' | CapProjectType;
+
+export type AppType =
+    | 'SAP Fiori elements'
+    | 'SAPUI5 freestyle'
+    | 'SAPUI5 Extension'
+    | 'Fiori Reuse'
+    | 'Fiori Adaptation';
```

Comment:
Const enum might be more convenient for the caller.

Reply:
@devinea mentioned this comment is not blocking and can be marked as resolved.